drivers/at86rf2xx/at86rfr2_interrupts.c

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (C) 2018 RWTH Aachen, Josua Arndt <jarndt@ias.rwth-aachen.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_at86rf2xx
+ * @{
+ *
+ * @file
+ * @brief       Implementation of public functions for AT86RF2xx drivers
+ *
+ * @author      Josua Arndt <jarndt@ias.rwth-aachen.de>
+ * @}
+ */
+/* As of now the implementation only supports the atmega256rfr2 */
+#ifdef MODULE_AT86RFR2
+
+#include "board.h"
+#include "avr/interrupt.h"
+#include "sched.h"
+
+#include "at86rf2xx_registers.h"
+#include "at86rf2xx_netdev.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#ifndef RXTX_LED_ENABLE
+    #define RXTX_LED_ENABLE (1)
+#endif
+
+#if RXTX_LED_ENABLE
+    #define RXTX_LED_INIT (LED_PORT_DDR |= LED0_MASK)
+    #define RXTX_LED_ON   LED0_ON
+    #define RXTX_LED_OFF  LED0_OFF
+#endif
+
+#define DEBUG_ATRFR2_PINS (0)
+#ifdef DEBUG_ATRFR2_PINS
+    #define DEBUG_ATRFR2_PORT_DDR     (DDRF)
+    #define DEBUG_ATRFR2_PORT         (PORTF)
+    #define DEBUG_ATRFR2_PIN_TX_START (1 << PORTF7)
+    #define DEBUG_ATRFR2_PIN_TX_END   (1 << PORTF6)
+    #define DEBUG_ATRFR2_PIN_RX_END   (1 << PORTF5)
+#endif
+
+/* netdev_t to at86rfr2_dev for Interrupt handling */
+netdev_t *at86rfr2_dev = 0;
+
+/* Counting Retries */
+int8_t retries = 0;
+
+void enable_rxtx_led(void)
+{
+
+#if RXTX_LED_ENABLE
+    RXTX_LED_INIT;
+#endif
+
+#if DEBUG_ATRFR2_PINS
+    /*initialize Debug Pins */
+    /* Port Pin as Output */
+    DEBUG_ATRFR2_PORT_DDR |=  (DEBUG_ATRFR2_PIN_TX_START
+                               | DEBUG_ATRFR2_PIN_TX_END | DEBUG_ATRFR2_PIN_RX_END);
+    /* Pin Low */
+    DEBUG_ATRFR2_PORT     &= ~(DEBUG_ATRFR2_PIN_TX_START
+                               | DEBUG_ATRFR2_PIN_TX_END | DEBUG_ATRFR2_PIN_RX_END);
+#endif
+}
+
+void _exit_isr_yield(void)
+{
+    if (sched_context_switch_request) {
+        thread_yield();
+        thread_yield_isr();
+    }
+    __exit_isr();
+}
+
+/**
+ * \brief ISR for transceiver's receive end interrupt
```

Comment:
For consistency in documentation, we prefer the use of `@brief`, here and below.